vendor/BuildWindowsGems.bat

```diff
@@ -21,88 +21,61 @@ IF [%1]==[] (
 ) else (
   set COSMOS_INSTALL=%~1
 )
-set MSYS64_32_PATH=C:\msys64
-set MSYS64_64_PATH=C:\msys64
-set QT_32_RUBY24_PATH=C:\Qt\4.8.6-Ruby24
-set Qt_64_RUBY24_PATH=C:\Qt\4.8.6-64-Ruby24
-set RUBY24_32_PATH=C:\Ruby242
-set RUBY24_64_PATH=C:\Ruby242-x64
-set RUBY25_32_PATH=C:\Ruby251
-set RUBY25_64_PATH=C:\Ruby251-x64
-
-:: Down to the main directory
+
+set CMAKE_PATH="C:\Program Files\CMake\bin"
+set QT_PATH=C:\Qt\4.8.6
+set RUBY24_64_PATH=C:\Ruby24-x64
+set RUBY25_64_PATH=C:\Ruby25-x64
+set RUBY26_64_PATH=C:\Ruby26-x64
+set MINGW_PATH=!RUBY26_64_PATH!\msys64\mingw64
```

Comment:
I assume it's ok to compile QT with Ruby 2.6 msys and use that with Ruby 2.4 and 2.5